app/src/main/res/values/strings.xml

```diff
@@ -168,10 +168,10 @@
     <string name="request_timeout">Request Timeout</string>
     <string name="superuser_notification">Superuser Notification</string>
     <string name="settings_su_reauth_title">Reauthenticate after upgrade</string>
-    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after an application upgrades</string>
-    <string name="settings_su_tapjack_title">Enable Tapjacking Protection</string>
+    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after upgrading apps</string>
+    <string name="settings_su_tapjack_title">Tapjacking Protection</string>
     <string name="settings_su_tapjack_summary">The superuser prompt dialog will not respond to input while obscured by any other window or overlay</string>
-    <string name="settings_su_biometric_title">Enable Biometric Authentication</string>
```

Comment:
This is a setting, so "enable" is implied.

Reply:
I agree removing the enable is probably fine. That's just me nitpicking. 😉